core/src/services/azblob/core.rs

```diff
@@ -41,7 +41,8 @@ use crate::*;
 mod constants {
     pub const X_MS_BLOB_TYPE: &str = "x-ms-blob-type";
     pub const X_MS_COPY_SOURCE: &str = "x-ms-copy-source";
-    pub const X_MS_BLOB_CONTENT_DISPOSITION: &str = "x-ms-blob-content-disposition";
+
+    pub const RESPONSE_CONTENT_DISPOSITION: &str = "rscd";
```

Comment:
This value will only be used once, please write them directly.

Reply:
Gotcha. Just pushed.